sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -71,7 +63,7 @@ gnrc_netif_t *gnrc_netif_create(char *stack, int stacksize, char priority,
                         _gnrc_netif_thread, (void *)netif, name);
     (void)res;
     assert(res > 0);
-    return netif;
+    return 0;
```

Comment:
always returns `0` so could be `void` instead or do you plan to add some error handling later on?

Reply:
I didn't want to make it void because that would remove all chances of checking error codes. There might be some cases where it still makes sense to use error codes (e.g MAC layers that only support one interface)

Reply:
agreed